libresapi/src/api/PeersHandler.cpp

```diff
@@ -106,6 +108,84 @@ bool peerInfoToStream(StreamBase& stream, RsPeerDetails& details, RsPeers* peers
     return ok;
 }
 
+std::string peerStateString(int peerState)
+{
+	if (peerState & RS_PEER_STATE_CONNECTED) {
+		return "Connected";
+	} else if (peerState & RS_PEER_STATE_UNREACHABLE) {
+		return "Unreachable";
+	} else if (peerState & RS_PEER_STATE_ONLINE) {
+		return "Available";
+	} else if (peerState & RS_PEER_STATE_FRIEND) {
+		return "Offline";
+	}
+
+	return "Neighbor";

```

Comment:
i'm not happy with sending user interface strings in json. But i don't have a better solution at the moment, so i have to accept it the way it is.
What we would need, is some kind of enum.